openfasoc/generators/gdsfactory-gen/tapeout_and_RL/opamp_perf_eval.sp

```diff
@@ -103,13 +103,14 @@ while bias_cs le bias_cs_Max
             ** Measure phase margin
             let phase = (180/PI)*vp(vo)
             meas ac pm find phase when vdb(vo)=0
+            let pm_FOM_factor = pm > 45 ? 1 : 0.0000001
```

Comment:
I'd say "-1" is a better factor for invalid PMs

Reply:
Suppose we have a case where none of the PM > 45.  If I made it just -1, we would essentially not be picking anything in the loop (where we are picking based on best FOM). This way we still pick the best FOM even though it may not be > 45. We at least get some data at the end